src/main/java/classify/user/InputParsing.java

```diff
@@ -18,6 +20,7 @@
 import java.util.logging.Logger;
 
 public class InputParsing {
+    private static final String EARLIER_POSSIBLE_DATE = "1970-01-01";
```

Comment:
Good use of magic literals, also good catch that the OS dates back to 1970-01-01!